src/app/dashboard/dashboard.component.scss

```diff
@@ -0,0 +1,98 @@
+@import 'src/app/styles/layout/variable.scss';
+
+//styles for the dashboard section.
+.dashboard {
+    height: 89vh;
+    overflow: auto;
+
+    //heading
+    .navbar-brand {
+        margin-bottom: 0;
+        color: $primary-color;
+        margin-left: 5px;
+    }
+    //button
+    .btn {
+        margin-right: 2px;
+    }
+
+    //container style for owe and owed section
+    .grid-container {
+        display: grid;
+        grid-template-columns: auto auto;
+        padding: 10px;
+        gap: 10px;
+        text-align: center;
+    }
+
+    .grid-item {
+        border: 1px solid $primary-color;
+        padding: 20px;
+        background-color: $secondary-color;
+        height: 400px;
+    }
+    
+    @media only screen and (max-width: 600px) {

```

Comment:
Media queries should at the end of normal style

Reply:
Changes done.